pom.xml

```diff
@@ -100,8 +100,8 @@
         <artifactId>maven-compiler-plugin</artifactId>
         <version>3.5.1</version>
         <configuration>
-          <source>1.7</source>
-          <target>1.7</target>
+          <source>1.8</source>
```

Comment:
I know you want to use all the new features, but if we ever want to use this with Android, I would prefer to leave this at 7. Maybe even get crazy and go back to 6.

Is there a reason you wanted to go to 8?